updater/lib/dependabot/api_client.rb

```diff
@@ -243,7 +245,7 @@ def record_ecosystem_versions(ecosystem_versions)
       rescue HTTP::ConnectionError, OpenSSL::SSL::SSLError
         retry_count ||= 0
         retry_count += 1
-        raise if retry_count > 3
+        raise if retry_count > MAX_REQUEST_RETRIES
 
```

Comment:
**Review Tip**: Moved retry request count in constant variable since it is always equal `3`.